sechub-doc/src/docs/asciidoc/documents/shared/configuration/sechub_config_example15_web_scan_header.json

```diff
@@ -0,0 +1,14 @@
+{
+  "apiVersion" : "1.0",
+  "webScan" : {
+    "url" : "https://productfailure.demo.example.org",
+    "headers" : [ {
+      "name" : "Authorization",
+      "value" : "{{ .HEADER_VALUE }}"
+    }, {
+      "name" : "x-file-size",
+      "value" : "123456",
+      "onlyForUrls" : [ "https://productfailure.demo.example.org/admin", "https://productfailure.demo.example.org/upload/<*>", "https://productfailure.demo.example.org/<*>/special/" ]
```

Comment:
You must mark the parts with the corresponding number via callouts. see  https://docs.asciidoctor.org/asciidoc/latest/verbatim/callouts/